match2/wikis/rocketleague/match_group_legacy_default.lua

```diff
@@ -7,13 +7,17 @@ local MAX_NUM_MAPS = 20
 
 local roundData
 function p.get(templateid, bracketType)
+	local LowerHeader = {}
 	local matches = mw.ext.Brackets.getCommonsBracketTemplate(templateid)
+
 	assert(type(matches) == "table")
 	local bracketData = {}
 	roundData = roundData or {}
+	local lastround = 0
 	for _, match in ipairs(matches) do
-		bracketData = p._getMatchMapping(match, bracketData, bracketType)
+		bracketData, lastround, LowerHeader = p._getMatchMapping(match, bracketData, bracketType, LowerHeader)
```

Comment:
```suggestion
		bracketData, lastRound, lowerHeader = p._getMatchMapping(match, bracketData, bracketType, LowerHeader)
```

Reply:
`lastRound` is already used (defined as a local of the module in line 42), so changing it here would produce issues
we probably should rename it to something else, maybe `lastR` or `finalRound`

Reply:
nvm, since the module wide one was renamed it is okay^^